tools/deploy_grafana.sh

```diff
@@ -161,15 +162,15 @@ do
   fi
 done
 
+sleep 20
```

Comment:
Why do we need to wait here?

Reply:
it takes some time for grafana handlers to be ready. I wasn't able to find an educated method to check when they are ready.
tried several methods. sleep seems to be the better one sadly.